src/site/content/en/mini-apps/mini-app-alternative-runtime-environments/index.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: post
+title: Other mini app runtime environments
+authors:
+  - thomassteiner
+date:  2021-02-05
+# updated: 2021-01-25
+description: |
+  This chapter presents a number of runtime environments for mini apps that are not mobile devices.
+tags:
+  - mini-apps
+---
+
+## More than just on mobile
+
+Apart from mobile devices, where mini apps are omnipresent and which are their natural habitat, mini
+apps have started to conquer other runtime environments like cars and the classic desktop.
+
+## Mini apps in cars
+
+In July 2020 the German car maker BMW Group
+[announced](https://www.press.bmwgroup.com/china/article/detail/T0313254ZH_CN/%E2%80%9C2020-%E5%AE%9D%E9%A9%AC%E7%A7%91%E6%8A%80%E6%97%A5%E2%80%9D%E5%9C%A8%E7%BA%BF%E5%8F%91%E5%B8%83%E5%A4%9A%E6%AC%BE%E8%BD%A6%E5%86%85%E6%95%B0%E5%AD%97%E4%BA%A7%E5%93%81)
+a collaboration with Tencent branded as WeScenario, which,
+[according to Tencent](https://www.tencent.com/en-us/articles/2201068.html), will be rolled out to _"30
+leading auto companies in the world, and [bring the WeScenario] ecosystem of social, content and
+services to more than 110 mainstream automobile models"_.
+
+<figure class="w-figure">
+  <img src="bmw-wescenario.jpg" alt="Dashboard of a Tencent car showing two rows of mini app icons.">
+  <figcaption class="w-figure">
+    Landing page of Tencent WeScenario (Source: <a href="https://www.press.bmwgroup.com/china/article/detail/T0313254ZH_CN/%E2%80%9C2020-%E5%AE%9D%E9%A9%AC%E7%A7%91%E6%8A%80%E6%97%A5%E2%80%9D%E5%9C%A8%E7%BA%BF%E5%8F%91%E5%B8%83%E5%A4%9A%E6%AC%BE%E8%BD%A6%E5%86%85%E6%95%B0%E5%AD%97%E4%BA%A7%E5%93%81">BMW</a>).
+  </figcaption>
+</figure>
+
+## Mini apps on the desktop
+
+### Mini apps in WeChat Desktop
+
+Using the WeChat desktop client available for
+[macOS](https://dldir1.qq.com/weixin/mac/WeChatMac.dmg)
+and [Windows](https://dldir1.qq.com/weixin/Windows/WeChatSetup.exe)), it is possible to run WeChat
+mini apps on the desktop. (Make sure to _not_ load the macOS
+[version from the App Store](https://itunes.apple.com/cn/app/id836500024), since it is more limited.)
```

Comment:
```suggestion
[version from the App Store](https://itunes.apple.com/cn/app/id836500024)
if you're doing research and want the full experience, since it is more limited.)
```
I feel like we should clarify a bit here